### ISSUE TITLE
std::vector<bool> causes castxml.exe to hang

### ISSUE THREAD

**Issue by taillonm** [CastXML/CastXML#50]
Hi, Thanks for your work. Here's my problem. I'm on windows 7 and tried with a few versions of Visual Studio / clang / LLVM:

    Class MyClass()
    {
    ...
      std::vector<bool> mBoolVector;
    };

When castxml is run on this class (-x c++ --castxml-gccxml --castxml-cc-msvc cl) , it hangs with the memory used by the process increasing rapidly. Have you seen this before ?

**Comment by bradking** [comment]
Using the snippet

```c++
#include <vector>
class MyClass
{
  std::vector<bool> mBoolVector;
};
```

I see no such problem.  Please provide a more complete source example.

**Comment by taillonm** [comment]
You gave me an idea. I tried your snippet and got the same result as you.

However, when I added -D_DEBUG to the command line. It hung like described previously.

**Comment by bradking** [comment]
Thanks.  Fixed by commit 8c3254367271184ef9b818171f09d262482e5301.
